### DISCUSSION TITLE
What is the purpose of CMGroup ?

### DISCUSSION THREAD

**Discussion by SergeStinckwich** [cormas/cormas#770] [Ideas]
``CMGroup`` is a subclass of ``CMAgent`` and an instance is composed of agents.

It is possible to describe why we need ``CMGroup`` class? Is there any model using this class?

**Comment by SergeStinckwich** [comment]
@pierbo answer on cormas-dev mailing-list: https://groups.google.com/d/msg/cormas-dev/4saw3XCdWjo/sT9GOauwAwAJ
Based on that, I will remove this class.

**Comment by SergeStinckwich** [comment]
@pierbo: I would suggest to delete the following classes: CMGroup, CMGroupLocation, CMGroupComm and CMGroupCommLocation.

**Comment by SergeStinckwich** [comment]
@pierbo any update on this issue?

**Comment by pierbo** [comment]
I thought we could delete this class, but I just discovered 2 people who were using it in their model!
So, I think we can leave the Agents hierarchy (with code repetition) for now (version 1.0) and think about the use of Traits for the next version, no?
